cookbook/case_studies/ml_training/house_price_prediction/README.rst

```diff
@@ -1,15 +1,17 @@
-Multiregion House Price Prediction Model Using XGBoost & Dynamic Workflows
---------------------------------------------------------------------------
-A house price prediction model is used to predict a house’s price, given the required parameters. Multiregion here relates to predicting the price of a house in multiple regions.
+House Price Regression
+-----------------------
+
+A house price prediction model is used to predict a house’s price, given the required inputs.
```

Comment:
```suggestion
A house price prediction model is used to predict a home's price, given the required inputs.
```

Reply:
if you don't mind I'll keep "house", because people buy houses, but they have to make it a home :)